src/beacon/app/main.py

```
@@ -27,30 +23,31 @@
 register_site_endpoints(app)
 register_data_analytics(app)
 
-
 # Custom JSON encoder to handle special values
 class CustomJSONEncoder(json.JSONEncoder):
     def default(self, obj):
         if isinstance(obj, Decimal):
-            # Convert Decimal to float
             try:
                 return float(obj)
             except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)